spec/recite_spec.rb

```diff
@@ -19,4 +34,32 @@
       "This is the horse and the hound and the horn that belonged to the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.")
     end
   end
+
+  describe "#random_recite" do
+    it "includes all phrases/lines" do
+      random_rhyme = recite_object.random_recite
+      known_lines.each do |known_line|
+        expect(random_rhyme).to include(known_line)
```

Comment:
suggestion: Using fixture files to compare against would be the cleaner way of testing the output of the recite methods.